api/src/main/java/org/openmrs/module/fhir2/providers/r3/MedicationRequestFhirResourceProvider.java

```diff
@@ -119,7 +120,9 @@ public IBundleProvider searchForMedicationRequests(
 	        @IncludeParam(allow = { "MedicationRequest:" + MedicationRequest.SP_MEDICATION,
 	                "MedicationRequest:" + MedicationRequest.SP_REQUESTER,
 	                "MedicationRequest:" + MedicationRequest.SP_PATIENT,
-	                "MedicationRequest:" + MedicationRequest.SP_CONTEXT }) HashSet<Include> includes) {
+	                "MedicationRequest:" + MedicationRequest.SP_CONTEXT }) HashSet<Include> includes,
+	        @IncludeParam(reverse = true, allow = {
+	                "MedicationDispense:" + MedicationDispense.SP_PRESCRIPTION }) HashSet<Include> revIncludes) {
```

Comment:
Note that MedicationDispense is not available in OpenMRS until 2.6.  If you try to make a request that includes this revinclude request on an earlier version of OpenMRS, you get a cryptic error. 

However, I didn't see a pattern to introduce a "MedicationRequestFhirResourceProvider2_6"... I'm willing to work through how to do this if we think this is a better practice.

Reply:
What kind of cryptic error @mogoodrich ?  I would think that - in pre-2.6 systems that the request to "get all associated MedicationDispenses" would just return an empty set, since none are there.  I don't see anything in this particular interface that is OpenMRS-version dependent.  If there is an issue, I assume it is in the implementation/services of getting and translation the MedicationDispenses, and I would assume we can just do a check there to return empty if they aren't supported?

Reply:
Good call @mseaton re: looking into the error... this may be an easy fix, will explore more tomorrow.